src/mlpack/tests/recurrent_network_test.cpp

```diff
@@ -328,28 +238,28 @@ void BatchSizeTest()
  */
 TEST_CASE("LSTMBatchSizeTest", "[RecurrentNetworkTest]")
 {
-  BatchSizeTest<LSTM<>>();
+  BatchSizeTest<Linear>();
 }
 
 /**
  * Ensure fast LSTMs work with larger batch sizes.
  */
-TEST_CASE("FastLSTMBatchSizeTest", "[RecurrentNetworkTest]")
-{
-  BatchSizeTest<FastLSTM<>>();
-}
+//TEST_CASE("FastLSTMBatchSizeTest", "[RecurrentNetworkTest]")
```

Comment:
Not sure should we remove the tests and add them later again?

Reply:
I moved them out to a `not_adapted/` directory in #3194---that'll probably work.  We can choose to re-enable them or delete them when adapting `FastLSTM`.

Reply:
Sounds good.